lib/osf-components/addon/components/copyable-text/styles.scss

```diff
@@ -1,4 +1,54 @@
 .CopyButton,
 .CopyableInput.CopyableInput {
```

Comment:
These extra items should just be fore the CopyButton. Fixing so that CopyableInput just has the height.